### PR TITLE
Fix documentation issues in JMS

### PR DESCRIPTION
**Pull request by aashikam**
## Purpose
Fixes documentation issues in JMS

### CODE REVIEW

File: stdlib/jms/src/main/ballerina/jms/durable_topic_subscriber.bal [hunk 1]
```diff
@@ -86,18 +92,18 @@ public type DurableTopicSubscriberEndpointConfiguration record {
...
-    # + return - error upon failure to acknowledge the received message
+    # + return - Error upon failure to acknowledge the received message
     public remote extern function acknowledge(Message message) returns error?;
 
     # Synchronously receive a message from the JMS provider
     #
-    # + timeoutInMilliSeconds - time to wait until a message is received
-    # + return - Returns a message or nill if the timeout exceededs. Returns an error on jms provider internal error.
+    # + timeoutInMilliSeconds - Time to wait until a message is received
+    # + return - Returns a message or nil if the timeout exceeds, returns an error upon jms provider internal error
```
Comment (samgnaniah) [inline stdlib/jms/src/main/ballerina/jms/durable_topic_subscriber.bal]: I think two sentences with the full stop was better here. Can we change that?

Change "jms provider" to "JMS Provider" as well.

File: stdlib/jms/src/main/ballerina/jms/message.bal [hunk 2]
```diff
@@ -23,19 +23,19 @@ public type Message object {
 
     # Gets text content of the JMS message
     #
-    # + return - the string containing this message's data or an JMS error
+    # + return - The string containing this message's data or an JMS error
```
Comment (samgnaniah) [inline stdlib/jms/src/main/ballerina/jms/message.bal]: Change "an JMS" to "a JMS". Apply everywhere.

File: stdlib/jms/src/main/ballerina/jms/topic_subscriber.bal [hunk 3]
```diff
@@ -108,14 +114,14 @@ public type TopicSubscriberCaller client object {
...
+    # + return - Returns a message or nil if the timeout exceeds, returns an error on jms provider internal error.
     public remote extern function receive(int timeoutInMilliSeconds = 0) returns (Message|error)?;
 
     # Synchronously receive a message from the JMS provider
     #
-    # + destination - destination to subscribe to
+    # + destination - Destination to subscribe to
     # + timeoutInMilliSeconds - Time to wait until a message is received
-    # + return - Returns a message or nill if the timeout exceededs. Returns an error on jms provider internal error.
+    # + return - Returns a message or nil if the timeout exceeds, returns an error on jms provider internal error
```
Comment (samgnaniah) [inline stdlib/jms/src/main/ballerina/jms/topic_subscriber.bal]: Same comment applies here.

File: stdlib/jms/src/main/ballerina/jms/simple_topic_subscriber.bal [hunk 4]
```diff
@@ -113,13 +119,14 @@ public type SimpleTopicSubscriber object {
...
-# + acknowledgementMode - JMS session acknwoledge mode
+# + initialContextFactory - JMS provider specific inital context factory
+# + providerUrl - JMS provider specific provider URL used to configure a connection
+# + connectionFactoryName - JMS connection factory to be used in creating JMS connections
+# + acknowledgementMode - Specifies the session mode that will be used. Legal values are "AUTO_ACKNOWLEDGE",
+#                         "CLIENT_ACKNOWLEDGE", "SESSION_TRANSACTED" and "DUPS_OK_ACKNOWLEDGE"
 # + messageSelector - Message selector condition to filter messages
-# + properties - JMS message properties
-# + topicPattern - Topic name pattern
+# + properties - Additional properties use in initializing the initial context
```
Comment (samgnaniah) [inline stdlib/jms/src/main/ballerina/jms/simple_topic_subscriber.bal]: Change "use in" to "used when".

File: stdlib/jms/src/main/ballerina/jms/durable_topic_subscriber.bal (hunk 1 above)
Comment (aashikam) [reply stdlib/jms/src/main/ballerina/jms/durable_topic_subscriber.bal]: All the other comments are without full stops. To make it consistent all the full stops are removed.

**Pull request merged**